test/encryption/encryption.test.js

```diff
@@ -33,8 +45,8 @@ describe('encryption integration tests', () => {
     const regex = /^(?<key>.*): "(?<value>.*)"$/;
     const variables = Object.fromEntries(file.map((line) => regex.exec(line.trim()).groups).map(({ key, value }) => [key, value]));
 
-    process.env.CRYPT_SHARED_LIB_PATH ??= variables.CRYPT_SHARED_LIB_PATH;
-    process.env.MONGOOSE_TEST_URI ??= variables.MONGODB_URI;
+    process.env.CRYPT_SHARED_LIB_PATH = variables.CRYPT_SHARED_LIB_PATH;
```

Comment:
I noticed locally that `MONGOOSE_TEST_URI` is often defined, resulting in the wrong URI being used.